tests/e2e/kserve_test.go

```diff
@@ -131,7 +135,7 @@ func (c *KserveTestCtx) validateFeatureTrackers(t *testing.T) {
 		jq.Match(`.metadata.ownerReferences[0].kind == "%s"`, gvk.Kserve.Kind),
 		jq.Match(`.metadata.ownerReferences[0].blockOwnerDeletion == true`),
 		jq.Match(`.metadata.ownerReferences[0].controller == true`),
-	))
+	), `Ensuring FeatureTracker %s has expected ownerReference`, ftName)
```

Comment:
These optional descriptions seem to force more useful output when a test fails (it doesn't always seem to output actual vs expected otherwise).